### PR TITLE
feat(dashboards): enable optional light mode for dashboards

### PR DESCRIPTION
**Pull request by alexpaxton**
- Create key in persisted state to determine how to render dashboards
- Write styles to override default (dark) mode when light mode is enabled
- Create a light theme for view types
- Add `Light/Dark` toggle to dashboard controls
- @121watts did some cleanup

Screen Shot 2020-03-12 at 1 15 43 PM
Screen Shot 2020-03-12 at 1 14 53 PM

- [ ] [CHANGELOG.md](https://github.com/influxdata/influxdb/blob/master/CHANGELOG.md) updated with a link to the PR (not the Issue)
- [x] [Well-formatted commit messages](https://www.conventionalcommits.org/en/v1.0.0-beta.3/)
- [x] Rebased/mergeable
- [x] Tests pass
- [ ] http/swagger.yml updated (if modified Go structs or API)
- [ ] Documentation updated or issue created (provide link to issue/pr)
- [ ] Signed [CLA](https://influxdata.com/community/cla/) (if not already signed)

### CODE REVIEW

**Comment by russorat** [comment]
@sanderson FYI

**Comment by sanderson** [comment]
Is this for dashboards only or the UI as a whole?

**Comment by alexpaxton** [comment]
@sanderson just dashboards for now. In our user research we learned that light mode was most useful for getting screenshots of a dashboard for use in presentations, or when the dashboard is on a wall mounted display

File: ui/src/App.tsx [hunk 1]
```diff
@@ -13,44 +15,61 @@ import CloudNav from 'src/pageLayout/components/CloudNav'
...
+      </AppWrapper>
+    </>
+  )
+}
 
 const mstp = (state: AppState): StateProps => {
   const {
     app: {
       ephemeral: {inPresentationMode},
+      persisted: {currentPage, theme},
```
Comment (ebb-tide) [inline ui/src/App.tsx]: I'm confused why we have a separate state for determining the current page- isn't it available via router path?

Comment (alexpaxton) [reply ui/src/App.tsx]: @121watts talked to @ebb-tide about our approach here a bit, and we concluded that it's unnecessary to have `currentPage` stored in persisted state. What do you think about moving it to `ephemeral` or somewhere else?

**Comment by 121watts** [comment]
@ebb-tide we don't need to persist the `currentPage` to local storage.  It can be stored in memory, specifically the redux store.

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,6 +2,7 @@
 
 ### Features
 
+1. [17232](https://github.com/influxdata/influxdb/pull/17232): Allow dashboards to optionally be displayed in light mode
```
Comment (ebb-tide) [inline CHANGELOG.md]: 👍

**Pull request merged**